OCP/Example4.md

```diff
@@ -0,0 +1,25 @@
+### Usage
+
+---
+#### Where
+
+https://github.com/melonjs/melonJS/blob/master/src/geometries/rectangle.js#L210-L214
+https://github.com/melonjs/melonJS/blob/master/src/renderable/renderable.js#L583-L587
```

Comment:
➕ ➖
I'm not sure about OCP in case we use a lot of different "if" statements based on different types and also instanceof.